From: Antonio Radici <antonio@debian.org>
Updated-By: наб <nabijaczleweli@nabijaczleweli.xyz>
Date: Tue, 15 Sep 2015 23:01:15 +0200
Updated-Date: Sat, 06 Nov 2021 14:23:53 +0100
Subject: smime.rc

Add CA's provided by ca-certificates to the ones
distributed by the default installation of Mutt.

Signed-off-by: Matteo F. Vescovi <mfv@debian.org>
---
 contrib/samples/smime.rc | 18 ++++++++++--------
 1 file changed, 10 insertions(+), 8 deletions(-)

diff --git a/contrib/samples/smime.rc b/contrib/samples/smime.rc
index 1c442c3..7df9da1 100644
--- a/contrib/samples/smime.rc
+++ b/contrib/samples/smime.rc
@@ -4,20 +4,20 @@
 
 # If you compiled neomutt with support for both PGP and S/MIME, PGP
 # will be the default method unless the following option is set
-set smime_is_default
+#set smime_is_default
 
 # Uncomment this if you don't want to set labels for certificates you add.
 # unset smime_ask_cert_label
 
 # Passphrase expiration
-set smime_timeout=300
+#set smime_timeout=300
 
 # Global crypto options -- these affect PGP operations as well.
-set crypt_auto_sign = yes
-set crypt_reply_encrypt = yes
-set crypt_reply_sign = yes
-set crypt_reply_sign_encrypted = yes
-set crypt_verify_sig = yes
+#set crypt_auto_sign = yes
+#set crypt_reply_encrypt = yes
+#set crypt_reply_sign = yes
+#set crypt_reply_sign_encrypted = yes
+#set crypt_verify_sig = yes
 
 
 # Section A: Key Management
@@ -47,9 +47,11 @@ set crypt_verify_sig = yes
 # unset smime_decrypt_use_default_key
 
 # Path to a file or directory with trusted certificates
-set smime_ca_location="~/.smime/ca-bundle.crt"
+# set smime_ca_location="~/.smime/ca-bundle.crt"
 
 # Path to where all known certificates go. (must exist!)
+set smime_ca_location=`find $HOME/.smime/ca-certificates.crt $HOME/.smime/ca-bundle.crt /etc/ssl/certs/ca-certificates.crt 2>/dev/null`
+
 set smime_certificates="~/.smime/certificates"
 
 # Path to where all private keys go. (must exist!)
